layouts/index.hbs

```diff
@@ -64,7 +64,7 @@
           </div>
         {{/if}}
         <p>
-          {{{ labels.version-schedule-prompt }}} <a href="https://github.com/nodejs/Release#release-schedule">{{ labels.version-schedule-prompt-link-text }}</a>
+          {{{ labels.version-schedule-prompt }}} <a href="http://nodejs.org/{{site.locale}}/about/releases/">{{ labels.version-schedule-prompt-link-text }}</a>
```

Comment:
Good thinking (forgot we have that link there), but the release page is not yet available in all languages.

Reply:
It's indeed a problem, is there any solution for this ? Or can you give me some advice to resolve this ?

Reply:
Well, the easiest solution would be to create the page in every language (copying from `en` for now).